packages/paste-core/utilities/grid/__tests__/grid.test.tsx

```diff
@@ -0,0 +1,588 @@
+  });
+
+  it('it should return vertical: 0', (): void => {
+    const mockVertical = false;
+    expect(getStackedColumns(mockVertical)).toStrictEqual('0');
+  });
+
+  it('it should return vertical: 100%', (): void => {
+    const mockVertical = true;
+    expect(getStackedColumns(mockVertical)).toStrictEqual('100%');
+  });
+
+  it('it should return vertical: 100%, 0, 100%', (): void => {
+    const mockVertical = [true, false, true];
+    expect(getStackedColumns(mockVertical)).toStrictEqual(['100%', '0', '100%']);
+  });
+
+  it('it should return vertical: null, null, null', (): void => {
+    const mockVertical = ['test', 'test', 'test'];
+    expect(getStackedColumns(mockVertical)).toStrictEqual([null, null, null]);
+  });
+
+  it('it should return offset: 16.666666666666664%', (): void => {
+    const mockOffset = 2;
+    expect(getColumnOffset(mockOffset)).toStrictEqual('16.666666666666664%');
+  });
+
+  it('it should return offset: 16.666666666666664%, 25%, 50%', (): void => {
+    const mockOffset = [2, 3, 6];
+    expect(getColumnOffset(mockOffset)).toStrictEqual(['16.666666666666664%', '25%', '50%']);
+  });
+
+  it('it should return span: 50%', (): void => {
+    const mockSpan = {
+      count: 12,
+      span: 6,
+    };
+    expect(getColumnSpan(mockSpan)).toStrictEqual('50%');
+  });
+
+  it('it should return span: 25%', (): void => {
+    const mockSpan = {
+      count: 12,
+      span: 3,
+    };
+    expect(getColumnSpan(mockSpan)).toStrictEqual('25%');
+  });
+
+  it('it should return span: 33.33333333333333%', (): void => {
+    const mockSpan = {
+      count: 3,
+    };
+    expect(getColumnSpan(mockSpan)).toStrictEqual('33.33333333333333%');
+  });
+
+  it('it should return span: 8.333333333333332%', (): void => {
+    const mockSpan = {
+      count: 12,
+    };
+    expect(getColumnSpan(mockSpan)).toStrictEqual('8.333333333333332%');
+  });
+
+  it('it should return span: 8.333333333333332% (null span)', (): void => {
+    const mockSpan = {
+      span: null,
+    };
+    expect(getColumnSpan(mockSpan)).toStrictEqual('8.333333333333332%');
+  });
+
+  it('it should return span: 25%, 50%, 25%', (): void => {
+    const mockSpan = {
+      count: 12,
+      span: [3, 6, 3],
+    };
+    expect(getColumnSpan(mockSpan)).toStrictEqual(['25%', '50%', '25%']);
+  });
+});
+
+describe('Grid Gutter Prop', () => {
+  it('it should not set a gutter property', (): void => {
+    const tree = renderer
+      .create(
+        <Theme.Provider theme="console">
+          <Grid>
+            <Column>
+              <Box backgroundColor="colorBackgroundPrimaryLight" width="100%" />
+            </Column>
+            <Column>
+              <Box backgroundColor="colorBackgroundPrimaryLight" width="100%" />
+            </Column>
+          </Grid>
+        </Theme.Provider>
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
```

Comment:
I don't think testing `it should not set a gutter property` should be done with a snapshot test because of two thoughts:
- This code is not deterministic (https://jestjs.io/docs/en/snapshot-testing#2-tests-should-be-deterministic)
- The same code should not be used for two different snapshot tests (this and line 154)